### PR TITLE
Help with a letter 

### PR DESCRIPTION
**Pull request by joaquinferroni**
i've finished the help button, available each hangman match

### CODE REVIEW

**Comment by huberalberti** [comment]
This code does not meet the standards of the team Joaco.
You will have to do it better next time
Request denied

File: app/src/main/res/layout/input_fragment.xml [hunk 1]
```diff
@@ -12,6 +12,20 @@
         android:layout_gravity="left|bottom">
 
         <TableRow>
+            <Button
```
Comment (lucasscarlatta) [inline app/src/main/res/layout/input_fragment.xml]: Move this into new fragment.xml file.

File: app/src/main/java/asd/org/ahorcado/activities/InputActivity.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
+/**
```
Comment (lucasscarlatta) [inline app/src/main/java/asd/org/ahorcado/activities/InputActivity.java]: This class is never used. Remove it.

File: app/src/main/java/asd/org/ahorcado/models/AbstractMatch.java [hunk 3]
```diff
@@ -45,6 +47,12 @@ public void execute(char letter) {
         }
     }
 
+    public abstract Map<String,String> showOneLetter();
+
+    /*public String getNewWord() {
```
Comment (lucasscarlatta) [inline app/src/main/java/asd/org/ahorcado/models/AbstractMatch.java:60]: Remove this comment.

File: app/src/main/java/asd/org/ahorcado/activities/MainActivity.java [hunk 4]
```diff
@@ -99,6 +103,26 @@ public void guessLetter(View view) {
...
+    }
+
+    public void showOneLetter(View view) {
+        Button buttonHelp=(Button) view;
+        buttonHelp.setEnabled(false);
+        Map result=gameController.showOneLetter();
+        guessLogic(view,(char)result.get("letter"));
+    }
+
+    private void disableLetterHelped(char letter) {
```
Comment (lucasscarlatta) [inline app/src/main/java/asd/org/ahorcado/activities/MainActivity.java:130]: Maybe you can move this into a new Fragment class.

File: app/src/main/java/asd/org/ahorcado/models/AbstractWord.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (dzcar) [inline app/src/main/java/asd/org/ahorcado/models/AbstractWord.java:1]: This class was removed a couple of commits before. Please update the branch

File: app/src/main/java/asd/org/ahorcado/models/Match.java [hunk 6]
```diff
@@ -3,9 +3,39 @@
...
+        Random r = new Random();
+        String partialWord=this.obtainPartialWord();
+        String completedWord=word.getWord();
+        char[] wordChars = partialWord.toCharArray();
+        char[] originalWordChars = this.word.getWord().toCharArray();
+        int positionLetter;
+        if (!partialWord.equals(completedWord)) {
+            do {
+                positionLetter = r.nextInt((wordChars.length - 1) - 0);
+            } while (wordChars[positionLetter] != '_');
```
Comment (dzcar) [inline app/src/main/java/asd/org/ahorcado/models/Match.java]: Please replace this '_' char with a constant. In order to do that, you should declare it as a constant in HangamanWord interface

File: .idea/gradle.xml [hunk 7]
```diff
@@ -3,9 +3,8 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
```
Comment (lucasscarlatta) [inline .idea/gradle.xml:4]: I update file .gitignore, so we don't need commit again this xml.

**Pull request merged**